test(testing) (#13)
Ermiass/crypto-tracking-app: README.md

<!-- PROJECT Header -->
<br />
<div align="center">

  <h2 align="center">CRYPTO-TRACKING-APP
</h3>

  <p align="center">
    Get All The Latest Info Regarding Your Favorite Crypto Currency And Crypto News Update!
    <br />
    <!-- <a href="https://github.com/othneildrew/Best-README-Template">View Demo</a> -->
</div>

<!-- TABLE OF CONTENTS -->
<details>
  <summary>Table of Contents</summary>
  <ol>
    <li>
      <a href="#about-the-project">About The Project</a>
      <ul>
        <li><a href="#built-with">Built With</a></li>
        <li><a href="#wireframe">Wireframe</a></li>
      </ul>
    </li>
    <li>
      <a href="#getting-started">Getting Started</a>
      <ul>
        <li><a href="#prerequisites">Prerequisites</a></li>
        <li><a href="#installation">Installation</a></li>
      </ul>
    </li>
    <li><a href="#usage">Usage</a></li>
     <li><a href="#contributing">Contributing</a></li>

>     <li><a href="#roadmap">Roadmap</a></li>

Reviewer: I believe this link points to a section that no longer exists; please remove.

    <li><a href="#contact">Contact</a></li>
  </ol>
</details>

<!-- ABOUT THE PROJECT -->
## About The Project

> This app is build with the existing knowledge of the React basics, and give you further practice working with React context,Chart Js, Firebase,MUI, codebase linting, component testing,Git-Flow workflow , and TypeScript.

Reviewer: Recommending the following spacing:

```suggestion
This app is build with the existing knowledge of the React basics, and give you further practice working with React context, Chart Js, Firebase, MUI, codebase linting, component testing, Git-Flow workflow, and TypeScript.
```

>  A user should, at the bare minimum, be able to view all the Top 100 list of Cryptocurrencys based on their market capital , Shows the trending coins over the last 24 hour,Search for a particular coin from the list and Shows a Historical chart of a specfic coin with a detail information on a daily ,monthly and year base and additionaly a user can learn and read on the latest update on the News section.

Reviewer: I'd recommend making the list of features/functionalities a bulleted list for easier readability.

For a specfic user with login  credential the user can add a coin to his the watch list and remove the coin from the watch list when ever he wantes.

> <p align="right">(<a href="#readme-top">back to top</a>)</p>

Reviewer: These 'back to top' links do not work.

### Built With

A few of the frameworks/libraries that were used to build the project are:

* React

> * Context ApI

Reviewer: ```suggestion
* Context API
```

* Chart Js
* Typescript
* MUI
* Firebase/Google authentication
* Axios
* React Router

> * React testing Libraries 

Reviewer: ```suggestion
* React Testing Library
```

> * EsLint AirBnb and Prettierr

Reviewer: Recommending typo fix:

```suggestion
* EsLint AirBnb and Prettier
```

<p align="right">(<a href="#readme-top">back to top</a>)</p>

> ### wireframe

Reviewer: Suggesting capitalization:

```suggestion
### Wireframe
```

  ### Getting Started The App build out with simple wireframes using Figma.
<img width="328" alt="wirefram-backend-blog-app" src="./public/wireframe.jpeg">

## Getting Started

To install CRYPTO-TRACKING-APP, follow these steps:
### Prerequisites

* npm
  sh
  npm install npm@latest -g
  

### Installation

> Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services.

Reviewer: This paragraph can be removed.

1. Clone the repo
   sh
   git clone https://github.com/Ermiass/crypto-tracking-app
   
3. Install NPM packages
   sh
   npm install
   
4. Get a free  API key from Firebase By Siging up and Enter your Firebase API_KEY link in your `.env` file 
   js
   REACT_APP_FIREBASE_API_KEY = 'ENTER YOUR API_KEY';
   
5. npm start 
  sh
   npm start 
   

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- USAGE EXAMPLES -->
## Usage

> Can be used to track the price of  top cryptocurencys, historical chart  and detail information on coin,get the latest news on cryptocurencys,cryptomarketes,NFTs and usfull teaching vidos.

Reviewer: Recommending some formatting/typo fixes:

```suggestion
Can be used to track the price of top cryptocurrencies, historical chart and detail information on coins, get the latest news on cryptocurrencies, cryptomarkets, NFTs and useful teaching videos.
```
